pages/index.tsx

```diff
@@ -1,16 +1,46 @@
 import React from "react";
-import { Spacer, Flex } from "@chakra-ui/react";
+import { Flex } from "@chakra-ui/react";
 
-import { Header, Main, Cards, Footer } from "@components";
+import SideContainer from "@components/sideSection/SideContainer";
+import ContentContainer from "@components/content/ContentContainer";
+import UploadButton from "@components/UploadButton";
+import TextAreaComponent from "@components/content/TextAreaComponent";
+import SideTitleComponent from "@components/shared/SideTitleComponent";
+import PreviewList from "@components/sideSection/PreviewList";
 
 const Home: React.FC = () => {
+  const renderLeftSide = () => (
```

Comment:
I guess later on we should refactor these 3 render functions to be a separate components - to make the code cleaner